test/reporter-html/diff.js

```diff
@@ -0,0 +1,163 @@
+QUnit.module( "diff" );
+
+QUnit.test( "throws if arguments are not strings", function( assert ) {
+	assert.throws(function() { QUnit.diff( {}, "" ); });
+	assert.throws(function() { QUnit.diff( "", {} ); });
```

Comment:
A practice I got after test262 where all cases were relevant, but that's about a language test. :)